docs/contracts/using-for.rst

```diff
@@ -16,17 +16,17 @@ at contract level.
 
 The first part, ``A``, can be one of:
 
-- a list of file-level or library functions (``using {f, g, h, L.t} for uint;``) -
-  only those functions will be attached to the type.
+- a list of file-level or internal library functions (``using {f, g, h, L.t} for uint;``) -
+  only those functions will be bound to the type as member functions,
```

Comment:
```suggestion
  only those functions will be bound to the type as member functions
```
Why is there a comma here? It's a list of bullet points, I don't think commas are necessary here.

Reply:
Well, I think that punctuation is recommended in lists. Though I guess normally it would be a semicolon or a full stop. I see now in most examples that it's more common to have nothing at the end and then a full stop for the last list item. Changed to that.